app/Providers/AppServiceProvider.php

```diff
@@ -34,7 +34,7 @@ class AppServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        if(!App::isLocal())
+        if(App::isProduction())
```

Comment:
why this we need to force this also on dev
that is why is asking if not local please roll it back

Reply:
I didn't change that. If you did for any reason please review.
Now is rolled back to if(!App::isLocal())